src/tools/miri/tests/compiletest.rs

```diff
@@ -88,6 +88,7 @@ fn test_config(target: &str, path: &str, mode: Mode, with_dependencies: bool) ->
         mode,
         program,
         output_conflict_handling,
+        out_dir: std::env::var_os("CARGO_TARGET_DIR").unwrap().into(),
```

Comment:
Will this put random files into the out_dir? Maybe it should be a `miri` subdir of that folder, or so.

Why does this work in Miri CI, is there some kind of default?

Reply:
yea there's a default. I need to fix this in ui_test specifically (should always append `ui` or sth)